Backend/src/database/services/team.service.ts

```diff
@@ -193,6 +193,11 @@ export default class TeamService {
     return await teamModel.find(filter).sort(sortBy);
   }
 
+  static async getUsersTeams(userId: (ObjectId | string)):
+      Promise<ITeam[] | null> {
+        return await teamModel.find({memberIds: {$elemMatch: {$eq: userId}}});
```

Comment:
Should also additionally search in adminIds
 ```suggestion
    return await teamModel.find({
      $or: [
        { memberIds: { $elemMatch: { $eq: userId } } },
        { adminIds: { $elemMatch: { $eq: userId } } },
      ],
    });
```